rustls/src/common_state.rs

```diff
@@ -51,6 +51,7 @@ pub struct CommonState {
     pub(crate) quic: quic::Quic,
     pub(crate) enable_secret_extraction: bool,
     temper_counters: TemperCounters,
+    pub(crate) refresh_traffic_keys_pending: Option<()>,
```

Comment:
Why `Option<()>` instead of just `bool`?

Reply:
I was fixated on wanting to write `.take()` but I guess I can use mem::take with a bool.